util/reggen/access.py

```diff
@@ -134,6 +154,9 @@ def needs_we(self) -> bool:
         '''
         return self.value[1] != SwAccess.RC and self.allows_write()
 
+    def to_systemrdl(self) -> dict():
```

Comment:
I know it's still a weak type, but can't we at least use something like `Dict[str, object]` to make it clear that the keys are strings?

Similarly with the `HwAccess` version below.